comp/core/workloadmeta/collectors/internal/kubelet/kubelet.go

```diff
@@ -313,6 +320,17 @@ func (c *collector) parsePodContainers(
 	return podContainers, events
 }
 
+func findPodGPUType(containers []kubelet.ContainerSpec) string {
```

Comment:
it would be nice if the function return a slice of string if never several GPU type as set on different container.

IMO we should add as many `kube_gpu_type:*` that we found. In most of the case it will always be 1, but at least the code will be more futur proof.

Reply:
Yeah that totally makes sense if the pod has containers requesting multiple different gpu resources then we should apply all of the tags accordingly. I didn't know that it was possible to do this with the tagger. Good to know!

Reply:
How would I go about adding multiple tags for kube_gpu_type. The current function description is:
```
// AddLow adds a new low cardinality tag to the list, or replace if already exists.
// It will skip empty values/names, so it's safe to use without verifying the value is not empty.
func (l *TagList) AddLow(name string, value string) {
	addTags(l.lowCardTags, name, value, l.splitList)
}
```
From what I understand from the description, it seems like if I were to run:
```
for _, GPUVendor := range pod.GPUVendorList {
     tagList.AddLow(tags.KubeGPUType, GPUVendor)
}
```
It seems like only the last GPUVendor value from the loop would persist bc it "replaces tags if it already exists"

Am I missing something?

Reply:
for public visibility, about the `tagList.AddLow(tags.KubeGPUType, GPUVendor)`  we looked a the implemented during our standup. the function `tagList.AddLow` can be call several time with the same key but different value. 
at the end the taglist contains as many tags than the different tag+value couple that we add.